argus/scenario.py

```diff
@@ -418,6 +418,61 @@ def get_remote_client(self, username=None, password=None,
     remote_client = util.cached_property(get_remote_client, 'remote_client')
 
 
+class RescueWindowsScenario(BaseWindowsScenario):
+    """Instance rescue Windows-based scenario."""
+
+    def rescue_server(self):
+        admin_pass = self._image.default_ci_password
+        self._servers_client.rescue_server(self._server['id'],
+                                           adminPass=admin_pass)
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'RESCUE')
+
+    def unrescue_server(self):
+        self._servers_client.unrescue_server(self._server['id'])
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'ACTIVE')
+
+    def run_test_suite(self):
```

Comment:
By the way, add a new entry in etc/argus.conf. That's the file we are using for production (with the image ids modified, of course).